scarb/tests/snforge_init.rs

```diff
@@ -3,14 +3,23 @@ use assert_fs::TempDir;
 use scarb::core::TomlManifest;
 use scarb_test_support::command::Scarb;
 use scarb_test_support::fsx::AssertFsUtf8Ext;
+use test_case::test_case;
 
-#[test]
+#[test_case(None)]
+#[test_case(Some("simple_project"))]
 #[ignore = "run this test by name"]
-fn new_simple() {
+fn new_simple(package_name: Option<&str>) {
```

Comment:
This test fails with (when run with `cargo test --profile=ci --package scarb --test snforge_init new_simple -- --ignored`): 
```
running 2 tests
test new_simple::none_expects ... FAILED
test new_simple::some_simple_project_expects ... FAILED

failures:

---- new_simple::none_expects stdout ----
thread 'new_simple::none_expects' panicked at scarb/tests/snforge_init.rs:34:5:
assertion failed: t.child("tests").is_dir()

---- new_simple::some_simple_project_expects stdout ----
thread 'new_simple::some_simple_project_expects' panicked at scarb/tests/snforge_init.rs:34:5:
assertion failed: t.child("tests").is_dir()
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace


failures:
    new_simple::none_expects
    new_simple::some_simple_project_expects

test result: FAILED. 0 passed; 2 failed; 0 ignored; 0 measured; 0 filtered out; finished in 1.58s
```

Note it's currently not run as part of our CI, it's only run once per day from main and during release -> https://github.com/software-mansion/scarb/blob/939c7f19e78708483b5e3230ffd03542337374d3/.github/workflows/_check-release.yml#L44-L50

Would you mind adding it to the CI pipeline as well? 🙏 https://github.com/software-mansion/scarb/blob/main/.github/workflows/ci.yml

Reply:
This should be fixed by the next foundry release, right?

Reply:
> This test fails with (when run with cargo test --profile=ci --package scarb --test snforge_init new_simple -- --ignored)

Are you sure you have the latest snfoundry version installed (`0.35.0`)? It was released yesterday

> Would you mind adding it to the CI pipeline as well?

Can we simply remove `#[ignore]` so it can be run with all other tests?

Reply:
> Can we simply remove #[ignore] so it can be run with all other tests?

The other tests run without Foundry installed.